### ISSUE TITLE
Would it be reasonable to move translations to another repo?

### ISSUE THREAD

**Issue by Gitoffthelawn** [WinMerge/winmerge#992]
I closely watch all code changes in this repo, but that causes an alert every time that any of the translations gets updated.

I'm happy that many of the translations are getting frequent updates, but would it be possible to move them to a different repo to reduce the number of notifications and alerts?

I'm pretty sure other repos do this, and if it would be helpful to find an example, I think I could find one for you.

TIA.

**Comment by sdottaka** [comment]
As of 9/29/2021, there are 11177 total git commits. Of these, the number of commits for changes to files in the Translations folder is 1080, about 1/10.

I don't think that's a lot.

Most translation changes are done via pull requests, so you may want to exclude pull requests from Watch.

**Comment by Gitoffthelawn** [comment]
@sdottaka Yep, GitHub just seems to notify on pull requests, not git commits, so maybe that's skewing it a lot.  It's not a big deal... just trying to go through my thousands of GitHub notifications and noticed a significant number of WinMerge translation notifications.  GitHub doesn't have a filtering mechanism when it's all in one repo,

The advantage to all the extra notifications is that I get that tiny bit of dopamine every time I click the "Done" button without having to read the issue.  The downside is all the time spent clicking that "Done" button for each translation issue. :)

**Comment by sdottaka** [comment]
I haven't tried it, but there seems to be a feature that allows you to exclude pull requests as follows

![image](https://user-images.githubusercontent.com/98126/135361700-26ffc33c-bc95-40ff-ad6b-57e4c19e4674.png)

![image](https://user-images.githubusercontent.com/98126/135361618-fc6a3473-a675-46ab-a2ff-1ca6b8d599aa.png)

**Comment by Gitoffthelawn** [comment]
Thanks.  I've tried it.  It works well.  But it excludes *all* pull requests from the repo.
